### PR TITLE
Neural network

### PR DESCRIPTION
**Pull request by gopichandpuli9**
I've implemented a neural network. The issue number is #47

### CODE REVIEW

File: Neural Network/neural-network.py [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+    def __init__(self, num_inputs, num_hidden, num_outputs, hidden_layer_weights = None, hidden_layer_bias = None, output_layer_weights = None, output_layer_bias = None):
+        self.num_inputs = num_inputs
+
+        self.hidden_layer = NeuronLayer(num_hidden, hidden_layer_bias)
+        self.output_layer = NeuronLayer(num_outputs, output_layer_bias)
+
+        self.init_weights_from_inputs_to_hidden_layer_neurons(hidden_layer_weights)
+        self.init_weights_from_hidden_layer_neurons_to_output_layer_neurons(output_layer_weights)
+
+    def init_weights_from_inputs_to_hidden_layer_neurons(self, hidden_layer_weights):
```
Comment (ashwani-rathee) [inline Neural Network/neural-network.py:32]: Keep the names of function smaller

**Review by ashwani-rathee — COMMENTED** [review]
@geekquad This stands invalid since it's completely plagiarized from https://github.com/mattm/simple-neural-network/blob/master/neural-network.py .Should I mark it invalid @gopichandpuli9?

**Comment by gopichandpuli9** [comment]
@ashwani-rathee I'm having exams. So I've copied the content. Sorry for that. You can assign the issue to someone who are willing to do. I'll start contributing once I'm done with my exams.

**Comment by ashwani-rathee** [comment]
@gopichandpuli9 Plagiarized

**Pull request closed without merging**